### ISSUE TITLE
#semantics configuration does not fully work

### ISSUE THREAD

**Issue by cmaeder** [UniFormal/MMT#121]
Our configuration resource contains:

```
#semantics
http://cds.omdoc.org/urtheories?LF info.kwarc.mmt.lf.Plugin
```

This is supposed to avoid explicitly loading the `lf.Plugin` extension as is done in #120. However, without the explicit extension `mmt.jar make mmt-omdoc --force` produces a different/wrong file
`content/http..cds.omdoc.org/urtheories/LFModulo.omdoc` an the errors shown in #100.

The error can be reproduced by:

`mmt.jar make mmt-omdoc --force source/openmath.mmt source/scala.mmt source/bifoundations.mmt source/meta.mmt source/module-expressions.mmt source/lf.mmt source/lfmodulo.mmt`

 However, the error is gone if either (the unrelated) `bifoundations` or `openmath` (or both) is/are omitted! (If `scala` is omitted then `bifoundations` fails.)

**Comment by cmaeder** [comment]
This is still a problem (after the fix of #120):

```
.../urtheories$ mmt.jar make mmt-omdoc --force source/bifoundations.mmt source/lfmodulo.mmt
    error: invalid unit: http://cds.omdoc.org/urtheories?Syntactic?[http://www.openmath.org/cd?OpenMath]/naryObject?definition:  |- List [Any] : /omitted_type
    error: invalid unit: http://cds.omdoc.org/urtheories?Syntactic?[http://www.openmath.org/cd?OpenMath]/binder?definition:  |- Fun [Context,Term]Any : /omitted_type
    error: invalid unit: http://cds.omdoc.org/urtheories?Semantic?[http://www.openmath.org/cd?OpenMath]/naryObject?definition:  |- List [Term] : /omitted_type
    error: invalid unit: http://cds.omdoc.org/urtheories?Semantic?[http://www.openmath.org/cd?OpenMath]/binder?definition:  |- Fun [Context,Term]Term : /omitted_type
    error: invalid unit: http://cds.omdoc.org/urtheories?LFModulo?equality?type:  |- {A::type}A→A→type INHABITABLE
    error: invalid unit: http://cds.omdoc.org/urtheories?LFModulo?equality?type:  |- {A::type}A→A→type INHABITABLE
    error: invalid unit: http://cds.omdoc.org/urtheories?LFModulo?equality?type:  |- {A::type}A→A→type INHABITABLE
```

**Comment by cmaeder** [comment]
the file `content/http..cds.omdoc.org/urtheories/LFModulo.omdoc` contains

```
<om:OMA><metadata><link rel="http://cds.omdoc.org/mmt?metadata?sourceRef" resource="http://docs.omdoc.org/urtheories/lfmodulo.mmt#85.3.16:89.3.20"/></metadata>
  <om:OMS base="http://cds.omdoc.org/urtheories" module="Typed" name="oftype"></om:OMS>
  <om:OMS base="http://cds.omdoc.org/urtheories" module="Typed" name="type">
    <metadata><link rel="http://cds.omdoc.org/mmt?metadata?sourceRef"
                         resource="http://docs.omdoc.org/urtheories/lfmodulo.mmt#86.3.17:89.3.20"/>
    </metadata>
</om:OMS>
</om:OMA>
```

instead of just:

```
<om:OMS base="http://cds.omdoc.org/urtheories" module="Typed" name="type">
  <metadata><link rel="http://cds.omdoc.org/mmt?metadata?sourceRef"
                             resource="http://docs.omdoc.org/urtheories/lfmodulo.mmt#86.3.17:89.3.20"/>
  </metadata>
</om:OMS>
```
